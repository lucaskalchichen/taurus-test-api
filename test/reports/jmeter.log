2025-05-18 21:55:40,118 INFO o.a.j.u.JMeterUtils: Setting Locale to en_US
2025-05-18 21:55:40,129 INFO o.a.j.JMeter: Loading user properties from: /root/.bzt/jmeter-taurus/5.5/bin/user.properties
2025-05-18 21:55:40,129 INFO o.a.j.JMeter: Loading system properties from: /root/.bzt/jmeter-taurus/5.5/bin/system.properties
2025-05-18 21:55:40,129 INFO o.a.j.JMeter: Loading additional properties from: /tmp/artifacts/jmeter-bzt.properties
2025-05-18 21:55:40,130 INFO o.a.j.JMeter: Setting System properties from file: /tmp/artifacts/system.properties
2025-05-18 21:55:40,133 INFO o.a.j.JMeter: Copyright (c) 1998-2022 The Apache Software Foundation
2025-05-18 21:55:40,134 INFO o.a.j.JMeter: Version 5.5
2025-05-18 21:55:40,134 INFO o.a.j.JMeter: java.version=11.0.27
2025-05-18 21:55:40,134 INFO o.a.j.JMeter: java.vm.name=OpenJDK 64-Bit Server VM
2025-05-18 21:55:40,134 INFO o.a.j.JMeter: os.name=Linux
2025-05-18 21:55:40,134 INFO o.a.j.JMeter: os.arch=amd64
2025-05-18 21:55:40,134 INFO o.a.j.JMeter: os.version=5.15.167.4-microsoft-standard-WSL2
2025-05-18 21:55:40,134 INFO o.a.j.JMeter: file.encoding=UTF-8
2025-05-18 21:55:40,135 INFO o.a.j.JMeter: java.awt.headless=true
2025-05-18 21:55:40,135 INFO o.a.j.JMeter: Max memory     =4160749568
2025-05-18 21:55:40,135 INFO o.a.j.JMeter: Available Processors =28
2025-05-18 21:55:40,138 INFO o.a.j.JMeter: Default Locale=English (United States)
2025-05-18 21:55:40,138 INFO o.a.j.JMeter: JMeter  Locale=English (United States)
2025-05-18 21:55:40,138 INFO o.a.j.JMeter: JMeterHome=/root/.bzt/jmeter-taurus/5.5
2025-05-18 21:55:40,138 INFO o.a.j.JMeter: user.dir  =/bzt-configs
2025-05-18 21:55:40,139 INFO o.a.j.JMeter: PWD       =/bzt-configs
2025-05-18 21:55:40,139 INFO o.a.j.JMeter: IP: 172.18.0.4 Name: bd039affccc7 FullName: bd039affccc7
2025-05-18 21:55:40,139 INFO o.a.j.JMeter: user.classpath=/tmp/artifacts:/tmp/artifacts
2025-05-18 21:55:40,139 INFO o.a.j.JMeter: Adding to classpath and loader: /tmp/artifacts
2025-05-18 21:55:40,141 INFO o.a.j.JMeter: Adding to classpath and loader: /tmp/artifacts
2025-05-18 21:55:40,148 INFO o.a.j.s.FileServer: Default base='/bzt-configs'
2025-05-18 21:55:40,148 INFO o.a.j.s.FileServer: Set new base='/tmp/artifacts'
2025-05-18 21:55:40,224 INFO o.a.j.s.SaveService: Testplan (JMX) version: 2.2. Testlog (JTL) version: 2.2
2025-05-18 21:55:40,234 INFO o.a.j.s.SaveService: Using SaveService properties version 5.0
2025-05-18 21:55:40,236 INFO o.a.j.s.SaveService: Using SaveService properties file encoding UTF-8
2025-05-18 21:55:40,237 INFO o.a.j.s.SaveService: Loading file: /tmp/artifacts/modified_requests.jmx
2025-05-18 21:55:40,257 INFO o.a.j.p.h.c.CacheManager: Will only cache the following methods: [GET]
2025-05-18 21:55:40,258 INFO o.a.j.p.h.c.CookieManager: Settings: Delete null: true Check: true Allow variable: true Save: false Prefix: COOKIE_
2025-05-18 21:55:40,266 INFO o.a.j.p.h.s.HTTPSamplerBase: Parser for text/html is org.apache.jmeter.protocol.http.parser.LagartoBasedHtmlParser
2025-05-18 21:55:40,266 INFO o.a.j.p.h.s.HTTPSamplerBase: Parser for application/xhtml+xml is org.apache.jmeter.protocol.http.parser.LagartoBasedHtmlParser
2025-05-18 21:55:40,266 INFO o.a.j.p.h.s.HTTPSamplerBase: Parser for application/xml is org.apache.jmeter.protocol.http.parser.LagartoBasedHtmlParser
2025-05-18 21:55:40,266 INFO o.a.j.p.h.s.HTTPSamplerBase: Parser for text/xml is org.apache.jmeter.protocol.http.parser.LagartoBasedHtmlParser
2025-05-18 21:55:40,266 INFO o.a.j.p.h.s.HTTPSamplerBase: Parser for text/vnd.wap.wml is org.apache.jmeter.protocol.http.parser.RegexpHTMLParser
2025-05-18 21:55:40,266 INFO o.a.j.p.h.s.HTTPSamplerBase: Parser for text/css is org.apache.jmeter.protocol.http.parser.CssParser
2025-05-18 21:55:40,283 INFO o.a.j.e.StandardJMeterEngine: Running the test!
2025-05-18 21:55:40,283 INFO o.a.j.s.SampleEvent: List of sample_variables: []
2025-05-18 21:55:40,283 INFO o.a.j.s.SampleEvent: List of sample_variables: []
2025-05-18 21:55:40,290 INFO o.a.j.e.u.CompoundVariable: Note: Function class names must contain the string: '.functions.'
2025-05-18 21:55:40,290 INFO o.a.j.e.u.CompoundVariable: Note: Function class names must not contain the string: '.gui.'
2025-05-18 21:55:40,452 INFO o.a.j.JMeter: Running test (1747605340452)
2025-05-18 21:55:40,462 INFO o.a.j.e.StandardJMeterEngine: Starting ThreadGroup: 1 : api-test
2025-05-18 21:55:40,462 INFO o.a.j.e.StandardJMeterEngine: Starting 1 threads for group api-test.
2025-05-18 21:55:40,462 INFO o.a.j.e.StandardJMeterEngine: Thread will continue on error
2025-05-18 21:55:40,464 INFO o.a.j.e.StandardJMeterEngine: All thread groups have been started
2025-05-18 21:55:40,464 INFO c.b.j.t.c.ConcurrencyThreadStarter: Start supplying threads
2025-05-18 21:55:41,981 INFO o.a.j.s.SampleResult: Note: Sample TimeStamps are START times
2025-05-18 21:55:41,981 INFO o.a.j.s.SampleResult: sampleresult.default.encoding is set to UTF-8
2025-05-18 21:55:41,982 INFO o.a.j.s.SampleResult: sampleresult.useNanoTime=true
2025-05-18 21:55:41,982 INFO o.a.j.s.SampleResult: sampleresult.nanoThreadSleep=5000
2025-05-18 21:55:41,982 INFO o.a.j.t.JMeterThread: Thread started: api-test-ThreadStarter 1-1
2025-05-18 21:55:41,999 INFO o.a.j.p.h.s.HTTPHCAbstractImpl: Local host = bd039affccc7
2025-05-18 21:55:42,003 INFO o.a.j.p.h.s.HTTPHC4Impl: HTTP request retry count = 0
2025-05-18 21:55:53,986 INFO c.b.j.t.c.ConcurrencyThreadStarter: Done supplying threads
2025-05-18 21:55:54,333 INFO o.a.j.t.JMeterThread: Thread is done: api-test-ThreadStarter 1-1
2025-05-18 21:55:54,333 INFO o.a.j.t.JMeterThread: Thread finished: api-test-ThreadStarter 1-1
2025-05-18 21:55:54,335 INFO o.a.j.e.StandardJMeterEngine: Notifying test listeners of end of test
2025-05-18 21:55:54,339 INFO o.a.j.e.StandardJMeterEngine: Forced JVM shutdown requested at end of test
