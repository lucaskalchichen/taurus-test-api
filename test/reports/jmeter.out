Created the tree successfully using /tmp/artifacts/modified_requests.jmx
Starting standalone test @ 2025 May 18 21:55:40 UTC (1747605340279)
Waiting for possible Shutdown/StopTestNow/HeapDump/ThreadDump message on port 4445
Tidying up ...    @ 2025 May 18 21:55:54 UTC (1747605354338)
... end of run
